.github/shared/src/changed-files.js

```diff
@@ -224,3 +239,17 @@ export function scenario(file) {
     typeof file === "string" && json(file) && specification(file) && file.includes("/scenarios/")
   );
 }
+
+/**
+ * Check if a path exists
+ * @param {string} path Path to check for existence
+ * @returns true if the path exists, false otherwise
+ */
+export async function pathExists(path) {
```

Comment:
This may belong somewhere else.

Reply:
If we need this, I'd move it to path.js.  But this is often an anti-pattern, where it's better to just try to read the file and handle exceptions.  Why do we need this?  Nobody is calling it in this PR.

Reply:
It's used in a couple places, notably https://github.com/Azure/azure-rest-api-specs/pull/35018/files#diff-12b87d3a9391806ac170a72a9edf0a80f908f1d9406c0d667783dd6612bd093cR105

Reply:
Or use `fs.existsSync()` from the standard lib.  `fs.exists()` is deprecated, but `fs.existsSync()` is still supported.

https://nodejs.org/api/fs.html#fsexistssyncpath